src/components/content/conditions/patient-conditions-all.tsx

```diff
@@ -0,0 +1,131 @@
+import cx from "classnames";
+import { useEffect, useMemo, useState } from "react";
+import { patientConditionsAllColumns } from "./helpers/columns";
+import { useConditionDetailsDrawer } from "./helpers/details";
+import { conditionFilters, defaultConditionFilters } from "./helpers/filters";
+import {
+  useAddConditionForm,
+  useConfirmDeleteCondition,
+  useEditConditionForm,
+} from "./helpers/modal-hooks";
+import { conditionSortOptions, defaultConditionSort } from "./helpers/sorts";
+import { ResourceTable } from "../resource/resource-table";
+import { ResourceTableActions } from "../resource/resource-table-actions";
+import { withErrorBoundary } from "@/components/core/error-boundary";
+import { useCTW } from "@/components/core/providers/use-ctw";
+import { RowActionsProps } from "@/components/core/table/table";
+import { ConditionModel } from "@/fhir/models";
+import { useFilteredSortedData } from "@/hooks/use-filtered-sorted-data";
+import { useBaseTranslations } from "@/i18n";
+import { usePatientConditionsAll } from "@/services/conditions";
+
+export type PatientConditionsAllProps = {
+  className?: string;
```

Comment:
Since this gets passed into `cx` it can use type `cx.Argument` which includes string but would also allow us to pass in anything valid to use with `cx`
```suggestion
  className?: cx.Argument;
```